src/components/Dropdown/Dropdown.tsx

```diff
@@ -102,43 +101,23 @@ const OptionsContainer = styled(Div)`
         : ''
     }
     z-index: 1000;
-    ${
-      isOpenedBelow
-        ? `
-          top: 100%;
-          border-top: 0px solid transparent;
-          border-radius: 0rem 0rem 0.25rem 0.25rem;
-        `
-        : `
-          top: auto;
-          bottom: 100%;
-          border-bottom: 0px solid transparent;
-          border-radius: 0.25rem 0.25rem 0rem 0rem;
-        `
-    }
-    ${isHidden ? `visibility: hidden;` : ''}
-  `}
+    ${direction}: 100%;
+    border-${direction}: 0px solid transparent;
+    border-radius: ${borderStyle};
+    ${isHidden ? 'visibility: hidden;' : ''}
+  `;
+  }}
 `;
 
 const HiddenOptionsContainer = styled(OptionsContainer)`
-  ${({ isOpenedBelow }) => `
+  ${({ isOpenedAbove }) => {
+    const direction = isOpenedAbove ? 'top' : 'bottom';
+
+    return `
     visibility: hidden;
-    height: 10rem;
-    ${
-      isOpenedBelow
-        ? `
-          top: 100%;
-          border-top: 0px solid transparent;
-          border-radius: 0rem 0rem 0.25rem 0.25rem;
-        `
-        : `
-          top: auto;
-          bottom: 100%;
-          border-bottom: 0px solid transparent;
-          border-radius: 0.25rem 0.25rem 0rem 0rem;
-        `
-    }
-  `}
+    ${direction}: 100%;
```

Comment:
if it's opened above, shouldn't the bottom be 100%?

Reply:
Good point, do you think `isOpenedAbove` should be renamed to `openDirection`?

Reply:
I think the name is okay - or it can just isOpenedBelow to be consistent with the other dropdown - but I feel like the direction is backwards, no?

Reply:
Yea it is a little backwards (just for naming, not functionality) since the hidden container is doing the opposite of what the options container does (I pass in !isOpenedBelow to this prop). I'll rename it to isOpenedBelow and add a comment where I pass in the prop saying the hidden container does the opposite of the options container.

Reply:
Can we just call it `isOpenedBelow` and pass it around as normal but then in the CSS just flip the ternary around